bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/internal/servlets/StaticImageGETServlet.java

```diff
@@ -49,27 +46,13 @@
     service = { Servlet.class },
     property = {
         "sling.servlet.resourceTypes=" + FormConstants.RT_FD_FORM_IMAGE_V1,
+        "sling.servlet.extensions=jpg",
+        "sling.servlet.extensions=png",
+        "sling.servlet.extensions=gif",
         "sling.servlet.methods=GET"
     })
 public class StaticImageGETServlet extends AbstractImageServlet {
```

Comment:
Why do we even need this servlet ? We changed the header component to use the image from WCM components. ideally, this image should also extend WCM component and we should just be adding fieldType like title or plain-text.